cmd/dep/status.go

```diff
@@ -1068,6 +1068,29 @@ func collectConstraints(ctx *dep.Ctx, p *dep.Project, sm gps.SourceManager) (con
 		}
 	}
 
+	// Incorporate constraints set in the manifest of the root project.
+	if p.Manifest != nil {
+
+		// Iterate through constraints in the manifest, append if it is a
+		// direct dependency
+		for pr, pp := range p.Manifest.Constraints {
```

Comment:
We can combine `p.Manifest.Constraints` and `p.Manifest.Ovr` into a single map, with override value taking precedence over any duplicate constraints entry, and iterate through overrides as well. Can do it as a follow-up PR.